website/atoma-node-optimization.md

```diff
@@ -0,0 +1,7 @@
+# Atoma vLLM
+
+It is possible to improve substantially Atoma's nodes performance for AI LLM inference. We are in the process of integrating vLLM and Paged Attention into Atoma's infrastructure, with the goal of having highly optimized memory management of incoming requests and continuously batching such requests for maximum GPU utilization.
+
+The way vLLM works is by scheduling incoming requests depending on the arrival order and of its size. Once a request is ready to be run, Atoma node can batched it with already scheduled previous requests and runs a single forward pass on multiple batched requests, being then able to generate the next prediction token of multiple requests simultaneously. 
```

Comment:
vLLM works by scheduling incoming requests depending on the arrival order and size. Once a request is ready to be run, Atoma node can batch it with previously scheduled requests and run a single forward pass on multiple batched requests. It is then able to generate the next prediction token of multiple requests simultaneously.

Reply:
Great !